.circleci/test.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+##
+# Run tests in CI.
+#
+set -e
+
+echo "==> Lint code"
+ahoy lint
+
+echo "==> Run Behat tests"
+mkdir -p /tmp/artifacts/behat
+ahoy cli "./vendor/bin/behat --strict --colors --tags="~@skipped" tests"
```

Comment:
The order matters a lot since some modules depend on others, which also makes it hard to fully isolate the tests. So please don't change the run order. more modules will be added soon.